tests/utils/throttleTick.test.js

```diff
@@ -0,0 +1,77 @@
+/* global assert, suite, test */
+var throttleTick = require('utils').throttleTick;
+
+suite('utils.throttleTick', function () {
+  var ts;
+  var dts;
+  var interval = 1000;
+  var throttleTickFn = throttleTick(function (t, dt) { ts.push(t); dts.push(dt); }, interval);
+
+  test('utils.throttleTick fires callback on first tick whatever', function () {
```

Comment:
test names don't have to all start with `utils.throttleTick` since that's the suite name. can be:

```fires first callback on first tick```